doc/cheatsheet.md

```diff
@@ -0,0 +1,259 @@
+    its special meaning.
+  * Backslash space = non-breaking space.
+  * Backslash at end of line = hard line break.
+
+## Links
+
+~~~
+<https://example.com>
+[read more](https://example.com)
+[read this too][foo bar]
+[one more link][]
+~~~
+
+then later:
+
+~~~
+[foo bar]: https://example.com
+[one more link]: https://example2.com
+~~~
+
+## Images
+
+~~~
+![beautiful skyline](clouds.jpg)
+![coastal shores][shore]
+![lush forests][]
+
+[shore]: the-beach.jpg
+[lush forests]: pines.jpg
+~~~
+
+## Headings
+
+~~~
+# H1
+## H2
+### H3
+~~~
+
+etc.
+
+## Lists
+
+~~~
+Itemized list:
+
+ * lions
+ * tigers
+ * bears
+~~~
+
+You can indent your list marker zero or more spaces, using the same
+indent for items at same level. If you want a subsequent paragraph to
+be part of a list item, indent at least past the list marker.
+
+Like subsequent paragraphs, sublists must be preceded by a blank line
+(blank lines are almost always required between block-level elements).
+The sublist markers must also be indented at least past the previous
+list item's marker.
+
+~~~ 
+Another list:
+
+ * First item. This item has a rather long
+   line, indented after the line breaks to
+   look nice.
+
+ * Second item. This item has a rather long
+line too, but without the pretty indent.
+ 
+   Second paragraph of second item.
+
+    - sublist,
+    - still in
+    - second item.
+
+ * Third item.
+
+Numbered list:
+
+ 1. item one
+ 2. item two
+ 3. item three
+
+Definition list:
+
+: lions
+
+  Like a C-program, these also have a main.
+  
+: tigers
+
+  Siberian, Bengal, and Tony the.
+  
+: bears
+
+  These come in both Teddy and Yogi.
+~~~
+
+## Tables
+
+~~~
+| Name | Size | Color |
+| --- | --- | --- |
+| lime | small | green |
+| orange | medium | orange |
+| grapefruit | large | yellow or pink |
+~~~
+
+You can alter the alignment of the cells using colons in the
+separator line, and also optionally add spaces to align the cells,
+e.g.:
+
+~~~
+| Material | Quantity | Catch-phrase  |
+| -------- | -------: | :-----------: |
+| cotton   |       42 |   Practical!  |
+| wool     |       17 |     Warm!     |
+| silk     |        4 |    Smooth!    |
+~~~
+
+
+## Footnotes
+
+~~~
+The proof is elementary.[^proof]
+
+...
+
+[^proof]: TODO. Haven't figured this out yet.
+~~~
+
+
+## Spans and Divs
+
+~~~
+This is [a span]{.some-class #some-id some-key="some val"}.
+~~~
+
+and here's a div:
+
+~~~
+{.some-class #some-other-id some-key="some val"}
+:::
+Div content
+goes here.
+:::
+~~~
+
+Note the following shorthand:
+
+~~~
+::: warning
+Watch out!
+:::
+~~~
+
+is equivalent to
+
+~~~
+{.warning}
+:::
+Watch out!
+:::
+~~~
+
+You can append attributes `_onto any inline_{.greenish}`,
+and prefix any block (including just an ordinary paragraph)
+with an attribute:
+
+~~~
+{.classy}
+This paragraph is wearing
+a top hat and cuff links.
+
+{source="personal-experience"}
+> More than three people on one
+> bicycle is *not* recommended.
+~~~
+
+
+## Raw Inline
```

Comment:
Since we have raw inlines and raw blocks, this title isn't quite appropriate.
Maybe "Raw (passthrough) content"?

Reply:
Ah. Thanks. Good catch. Corrected the heading and added a sentence after to clarify.